ext/js/language/text-scanner.js

```diff
@@ -87,6 +87,9 @@ class TextScanner extends EventDispatcher {
         this._pointerIdTypeMap = new Map();
 
         this._canClearSelection = true;
+
+        this._yomichanIsChangingTextSelectionNow = false;
+        this._userHasNotSelectedAnythingManually = true;
```

Comment:
Reset the state of these variables in `setEnabled`; you may also want to assign `_userHasNotSelectedAnythingManually` based on the current selection in there if `value` is true. This would handle the following edge case:

1. User has Yomichan disabled
2. Opens webpage
3. Selects text
4. Enables Yomichan via the browser action popup
5. Scans text
6. User selection is lost

Reply:
Aha! Will do.